packages/react/polymorphic/README.md

```diff
@@ -10,4 +13,4 @@ $ npm install @radix-ui/react-polymorphic
 
 ## Usage
 
-This is an internal utility, not intended for public usage.
+See [the docs](https://radix-ui.com/primitives/docs/utilities/polymorphic).
```

Comment:
For consistency with our other packages, we usual write the following here:

```suggestion
View docs [here](https://radix-ui.com/primitives/docs/utilities/polymorphic).
```

Just in case we ever decide to do a project wide find & replace 🙏

Reply:
Resolved locally, closing this :)